frontend/Caddyfile

```diff
@@ -23,7 +23,7 @@
 		Cache-Control "no-store, no-cache, must-revalidate, proxy-revalidate"
 		X-Content-Type-Options "nosniff"
 		Strict-Transport-Security "max-age=31536000"
-		Content-Security-Policy "base-uri 'self'; connect-src 'self' https://*.gov.bc.ca https://*.amazoncognito.com https://*.cloudfront.net https://cdnjs.cloudflare.com/ https://cognito-idp.ca-central-1.amazonaws.com https://results-ques.netlify.app/api; default-src 'self'; font-src 'self' https://dq9dozspu8y40.cloudfront.net/20230404192231/; frame-src 'self' https://*.gov.bc.ca; img-src 'self' http://www.w3.org/ https://cdnjs.cloudflare.com/ https://www.google.ca/maps/ https://*.tile.openstreetmap.org/; manifest-src 'self'; media-src 'self'; object-src 'none'; script-src 'unsafe-inline' 'report-sample' 'self' https://cdn.jsdelivr.net/npm/; script-src-elem 'unsafe-inline' 'report-sample' 'self' https://cdn.jsdelivr.net/npm/ https://dq9dozspu8y40.cloudfront.net/20230404192231/; style-src 'report-sample' 'self' https://cdn.jsdelivr.net/npm/ https://dq9dozspu8y40.cloudfront.net/; style-src-elem 'report-sample' 'self' https://cdn.jsdelivr.net/npm/ http://cdnjs.cloudflare.com/ https://dq9dozspu8y40.cloudfront.net/20230404192231/; worker-src 'none';"
+		Content-Security-Policy "base-uri 'self'; connect-src 'self' https://*.gov.bc.ca https://*.amazoncognito.com https://*.cloudfront.net https://cdnjs.cloudflare.com/ https://cognito-idp.ca-central-1.amazonaws.com https://results-ques.netlify.app/api; default-src 'self'; font-src 'self' https://dq9dozspu8y40.cloudfront.net/20230404192231/; frame-src 'self' https://*.gov.bc.ca; img-src * 'self' blob: data: https: http://www.w3.org/ https://cdnjs.cloudflare.com/ https://www.google.ca/maps/ https://*.tile.openstreetmap.org/; manifest-src 'self'; media-src 'self'; object-src 'none'; script-src 'unsafe-inline' 'report-sample' 'self' https://cdn.jsdelivr.net/npm/; script-src-elem 'unsafe-inline' 'report-sample' 'self' https://cdn.jsdelivr.net/npm/ https://dq9dozspu8y40.cloudfront.net/20230404192231/; style-src 'report-sample' 'self' https://cdn.jsdelivr.net/npm/ https://dq9dozspu8y40.cloudfront.net/; style-src-elem 'report-sample' 'self' https://cdn.jsdelivr.net/npm/ http://cdnjs.cloudflare.com/ https://dq9dozspu8y40.cloudfront.net/20230404192231/; worker-src 'none';"
```

Comment:
Block here!  Just so we remember to remove this before merging: `img-src * 'self' blob: data: https:`